<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnDelMaster">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除Master</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_masters">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddMaster">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>增加Master</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrint">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>print masters</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_addMaster" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>291</width>
     <height>201</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_IP">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Port">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>193</width>
      <height>28</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Name">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>M点：   M2</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
